src/libultra_code/coss.c

```diff
@@ -1,3 +1,6 @@
 #include "libultra_internal.h"
+#include <functions.h>
```

Comment:
```suggestion
#include <global.h>
```
Does it work to just include global.h? Just for consistency. Other than that minor comment, looks good.